lib/Mojolicious/Command/nopaste/Service/fpaste.pm

```diff
@@ -0,0 +1,35 @@
+package Mojolicious::Command::nopaste::Service::fpaste;
+use Mojo::Base 'Mojolicious::Command::nopaste::Service';
+
+has description => "Post to fpaste.org\n";
+
+sub paste {
+  my $self = shift;
+  my $lang = $self->language || 'text';
```

Comment:
I was going by what I read in `debian.pm` and `pastie.pm` which use text if `$self->language` does not return a true value. Just pointing that out - but I will happily change to `my $lang = $self->language || 'perl'` if desired.